### ISSUE TITLE
Resolve contrast accessibility violations

### ISSUE THREAD

**Issue by lawhorkl** [uclibs/scholar_uc#1587]
### Descriptive summary

Our recent accessibility scan of Scholar 3 reveals 105 total accessibility issues, 83 of these are contrast issues between the background and foreground colors.

We need to resolve them before we push to prod.

To new developers: Do not be intimidated by the list length, it is most likely the case that a few small CSS changes will resolve most if not all of the issues listed.

A good tool while working on this is https://achecker.ca/checker/index.php to allow you to make sure your changes solve the issue.

The following is a list of all contrast accessibility issues present.

* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044361
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044362
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044363
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044364
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044365
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044366
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044367
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044368
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044369
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044370
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044388
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044389
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044390
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044391
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044392
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044393
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044395
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044396
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044397
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044398
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044399
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044400
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044403
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044404
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044405
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044406
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044407
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044408
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044411
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044412
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044413
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044414
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044415
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044416
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044581
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044582
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044583
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044584
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044585
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044586
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044747
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044748
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044749
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044750
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044751
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044752
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044753
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044754
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044755
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044756
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044757
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044758
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044759
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044760
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044947
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044948
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044949
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044950
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044951
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044952
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044953
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045084
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045085
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045086
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045087
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045088
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045255
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045256
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045257
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045258
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045259
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045406
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045407
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045408
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045409
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045410
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045556
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045557
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045558
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045559
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045560
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044372
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045412
